src/OpenApi/test/Microsoft.AspNetCore.OpenApi.Tests/Integration/snapshots/OpenApi3_0/OpenApiDocumentIntegrationTests.VerifyOpenApiDocument_documentName=schemas-by-ref.verified.txt

```diff
@@ -507,6 +507,28 @@
           }
         }
       }
+    },
+    "/schemas-by-ref/json-patch": {
+      "patch": {
+        "tags": [
+          "Sample"
+        ],
+        "requestBody": {
+          "content": {
+            "application/json-patch+json": {
+              "schema": {
+                "$ref": "#/components/schemas/JsonPatchDocumentOfParentObject"
```

Comment:
It seems like we need a custom JSON convert to model the schema for JSON path document. I assume the schema looks something like:

```
{
  type: "object",
  properties: {
     "op": "string",
      "path": "string",
      "value": "string"
  }
}
```

@mikekistler Sanity checking this by you. Does it seem sufficient to use the same schema for all JsonPatchDocument types or should we figure out some way to capture the properties of the model that you're patchinmg in the schema?

Reply:
If needed I could look into doing that in a follow-up, as that's existing behaviour anyway without the media type change.

Reply:
Actually, the `JsonPatchDocumentOfParentObject` is just this (line 624):

```json
 "JsonPatchDocumentOfParentObject": { },
```

I think we can tackle the issue of the patch body schema in a separate PR. We should probably open an issue for this. I don't have an issue with using the same schema for all patch operations, but it should be more like the schema you show above.